docs/application/native/guides/notification/notification-ex.md

```diff
@@ -0,0 +1,532 @@
+    noti_ex_item_h button_item = NULL;
+    noti_ex_item_h selector_item = NULL;
+
+    ret = noti_ex_item_image_create(&image_item_icon, "icon_id", icon_path);
+    ret = noti_ex_item_image_create(&image_item1, "image_id1", image1_path);
+    ret = noti_ex_item_image_create(&image_item2, "image_id2", image2_path);
+    ret = noti_ex_item_text_create(&title_item, "title_id", "Jone", NULL);
+    ret = noti_ex_item_text_create(&contents_item, "contents_id", "How are you ...", NULL);
+    ret = noti_ex_item_button_create(&button_item, "button_id", "button_title");
+    ret = noti_ex_item_input_selector_create(&selector_item, "selector_id");
+    const char *contents[] = {"What's up?", "Nice",. "On my way"};
+    ret = noti_ex_item_input_selector_set_contents(selector_item, contents, 3);
+
+    ret = noti_ex_item_group_create(&group_item, "group_id");
+    ret = noti_ex_item_group_add_child(group_item, image_item_icon);
+    ret = noti_ex_item_group_add_child(group_item, title_item);
+    ret = noti_ex_item_group_add_child(group_item, contents_item);
+    ret = noti_ex_item_group_add_child(group_item, image_item1);
+    ret = noti_ex_item_group_add_child(group_item, image_item2);
+    ret = noti_ex_item_group_add_child(group_item, button_item);
+    ret = noti_ex_item_group_add_child(group_item, selector);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+<a name="messaging"></a>
+## Creating Messaging style Notification
+
+Since messages with different values can be added to a single notification, you can either show various messages in a single notification or you can configure it like a chat-room.
+You can also configure notifications using text items, image items, and time items.
+
+**Figure: Messaging style Notification**
+
+![Messaging style Notification](./media/messaging_notification.png)
+
+    ```
+    noti_ex_item_h group_item = NULL;
+    noti_ex_item_h chatmessage_item = NULL;
+    noti_ex_item_h name_handle = NULL;
+    noti_ex_item_h text_handle = NULL;
+    noti_ex_item_h image_handle = NULL;
+    noti_ex_item_h time_handle = NULL;
+    time_t current_time;
+
+    ret = noti_ex_item_group_create(&group_item, "group_id");
+
+    noti_ex_item_text_create(&name_handle, "name_id", "name", NULL);
+    noti_ex_item_text_create(&text_handle, "text_id", "text", NULL);
+    noti_ex_item_image_create(&image_handle, "image_id", APP_IMAGE_FULL_PATH);
+
+    time(&current_time);
+    noti_ex_item_time_create(&time_handle, "time_id", current_time);
+
+    ret = noti_ex_item_chat_message_create(&chatmessage_item, "message_id", name_handle,
+        text_handle, image_handle, time_handle, NOTI_EX_ITEM_CHAT_MESSAGE_TYPE_USER);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+
+    ret = noti_ex_item_group_add_child(group_item, chatmessage_item);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+<a name="reporter"></a>
+## Make a Notification-ex Reporter Handle
```

Comment:
Please consider https://github.com/Samsung/tizen-docs/pull/887#discussion_r336356958

Reply:
Can you tell me again? I can't find.

Reply:
@mk5004lee 
1. It is recommended to use a gerund form (ing form) heading for Tizen docs. 
2. In technical documents, it is better to use "create" than "make".
3. It is recommended to avoid articles if possible in a heading
Considering the above three points, request you to change the title to:

## Creating Notification-ex Reporter Handle

Please let me know if this is not clear.